### ISSUE TITLE
Why the params of MobileDets in Jetson Xavier GPU is so large?

### ISSUE THREAD

**Issue by taojianggit** [novice03/mobiledet-pytorch#1]
`----------------------------------------------------------------
        Layer (type)               Output Shape         Param #
================================================================
            Conv2d-1         [-1, 32, 219, 220]             896
       BatchNorm2d-2         [-1, 32, 219, 220]              64
             ReLU6-3         [-1, 32, 219, 220]               0
            Conv2d-4          [-1, 8, 219, 220]             256
       BatchNorm2d-5          [-1, 8, 219, 220]              16
             ReLU6-6          [-1, 8, 219, 220]               0
            Conv2d-7         [-1, 16, 219, 220]           1,152
       BatchNorm2d-8         [-1, 16, 219, 220]              32
             ReLU6-9         [-1, 16, 219, 220]               0
           Conv2d-10         [-1, 16, 219, 220]             256
      BatchNorm2d-11         [-1, 16, 219, 220]              32
       TuckerConv-12         [-1, 16, 219, 220]               0
           Conv2d-13        [-1, 128, 110, 110]          18,432
      BatchNorm2d-14        [-1, 128, 110, 110]             256
            ReLU6-15        [-1, 128, 110, 110]               0
         Identity-16        [-1, 128, 110, 110]               0
           Conv2d-17         [-1, 32, 110, 110]           4,096
      BatchNorm2d-18         [-1, 32, 110, 110]              64
     EdgeResidual-19         [-1, 32, 110, 110]               0
           Conv2d-20          [-1, 8, 110, 110]             256
      BatchNorm2d-21          [-1, 8, 110, 110]              16
            ReLU6-22          [-1, 8, 110, 110]               0
           Conv2d-23          [-1, 8, 110, 110]             576
      BatchNorm2d-24          [-1, 8, 110, 110]              16
            ReLU6-25          [-1, 8, 110, 110]               0
           Conv2d-26         [-1, 32, 110, 110]             256
      BatchNorm2d-27         [-1, 32, 110, 110]              64
       TuckerConv-28         [-1, 32, 110, 110]               0
           Conv2d-29          [-1, 8, 110, 110]             256
      BatchNorm2d-30          [-1, 8, 110, 110]              16
            ReLU6-31          [-1, 8, 110, 110]               0
           Conv2d-32          [-1, 8, 110, 110]             576
      BatchNorm2d-33          [-1, 8, 110, 110]              16
            ReLU6-34          [-1, 8, 110, 110]               0
           Conv2d-35         [-1, 32, 110, 110]             256
      BatchNorm2d-36         [-1, 32, 110, 110]              64
       TuckerConv-37         [-1, 32, 110, 110]               0
           Conv2d-38          [-1, 8, 110, 110]             256
      BatchNorm2d-39          [-1, 8, 110, 110]              16
            ReLU6-40          [-1, 8, 110, 110]               0
           Conv2d-41          [-1, 8, 110, 110]             576
      BatchNorm2d-42          [-1, 8, 110, 110]              16
            ReLU6-43          [-1, 8, 110, 110]               0
           Conv2d-44         [-1, 32, 110, 110]             256
      BatchNorm2d-45         [-1, 32, 110, 110]              64
       TuckerConv-46         [-1, 32, 110, 110]               0
           Conv2d-47          [-1, 256, 55, 55]          73,728
      BatchNorm2d-48          [-1, 256, 55, 55]             512
            ReLU6-49          [-1, 256, 55, 55]               0
         Identity-50          [-1, 256, 55, 55]               0
           Conv2d-51           [-1, 64, 55, 55]          16,384
      BatchNorm2d-52           [-1, 64, 55, 55]             128
     EdgeResidual-53           [-1, 64, 55, 55]               0
           Conv2d-54          [-1, 512, 55, 55]         294,912
      BatchNorm2d-55          [-1, 512, 55, 55]           1,024
            ReLU6-56          [-1, 512, 55, 55]               0
         Identity-57          [-1, 512, 55, 55]               0
           Conv2d-58           [-1, 64, 55, 55]          32,768
      BatchNorm2d-59           [-1, 64, 55, 55]             128
     EdgeResidual-60           [-1, 64, 55, 55]               0
           Conv2d-61          [-1, 512, 55, 55]         294,912
      BatchNorm2d-62          [-1, 512, 55, 55]           1,024
            ReLU6-63          [-1, 512, 55, 55]               0
         Identity-64          [-1, 512, 55, 55]               0
           Conv2d-65           [-1, 64, 55, 55]          32,768
      BatchNorm2d-66           [-1, 64, 55, 55]             128
     EdgeResidual-67           [-1, 64, 55, 55]               0
           Conv2d-68          [-1, 256, 55, 55]         147,456
      BatchNorm2d-69          [-1, 256, 55, 55]             512
            ReLU6-70          [-1, 256, 55, 55]               0
         Identity-71          [-1, 256, 55, 55]               0
           Conv2d-72           [-1, 64, 55, 55]          16,384
      BatchNorm2d-73           [-1, 64, 55, 55]             128
     EdgeResidual-74           [-1, 64, 55, 55]               0
           Conv2d-75          [-1, 512, 28, 28]         294,912
      BatchNorm2d-76          [-1, 512, 28, 28]           1,024
            ReLU6-77          [-1, 512, 28, 28]               0
         Identity-78          [-1, 512, 28, 28]               0
           Conv2d-79          [-1, 128, 28, 28]          65,536
      BatchNorm2d-80          [-1, 128, 28, 28]             256
     EdgeResidual-81          [-1, 128, 28, 28]               0
           Conv2d-82          [-1, 512, 28, 28]         589,824
      BatchNorm2d-83          [-1, 512, 28, 28]           1,024
            ReLU6-84          [-1, 512, 28, 28]               0
         Identity-85          [-1, 512, 28, 28]               0
           Conv2d-86          [-1, 128, 28, 28]          65,536
      BatchNorm2d-87          [-1, 128, 28, 28]             256
     EdgeResidual-88          [-1, 128, 28, 28]               0
           Conv2d-89          [-1, 512, 28, 28]         589,824
      BatchNorm2d-90          [-1, 512, 28, 28]           1,024
            ReLU6-91          [-1, 512, 28, 28]               0
         Identity-92          [-1, 512, 28, 28]               0
           Conv2d-93          [-1, 128, 28, 28]          65,536
      BatchNorm2d-94          [-1, 128, 28, 28]             256
     EdgeResidual-95          [-1, 128, 28, 28]               0
           Conv2d-96          [-1, 512, 28, 28]         589,824
      BatchNorm2d-97          [-1, 512, 28, 28]           1,024
            ReLU6-98          [-1, 512, 28, 28]               0
         Identity-99          [-1, 512, 28, 28]               0
          Conv2d-100          [-1, 128, 28, 28]          65,536
     BatchNorm2d-101          [-1, 128, 28, 28]             256
    EdgeResidual-102          [-1, 128, 28, 28]               0
          Conv2d-103         [-1, 1024, 28, 28]       1,179,648
     BatchNorm2d-104         [-1, 1024, 28, 28]           2,048
           ReLU6-105         [-1, 1024, 28, 28]               0
        Identity-106         [-1, 1024, 28, 28]               0
          Conv2d-107          [-1, 128, 28, 28]         131,072
     BatchNorm2d-108          [-1, 128, 28, 28]             256
    EdgeResidual-109          [-1, 128, 28, 28]               0
          Conv2d-110         [-1, 1024, 28, 28]       1,179,648
     BatchNorm2d-111         [-1, 1024, 28, 28]           2,048
           ReLU6-112         [-1, 1024, 28, 28]               0
        Identity-113         [-1, 1024, 28, 28]               0
          Conv2d-114          [-1, 128, 28, 28]         131,072
     BatchNorm2d-115          [-1, 128, 28, 28]             256
    EdgeResidual-116          [-1, 128, 28, 28]               0
          Conv2d-117         [-1, 1024, 28, 28]       1,179,648
     BatchNorm2d-118         [-1, 1024, 28, 28]           2,048
           ReLU6-119         [-1, 1024, 28, 28]               0
        Identity-120         [-1, 1024, 28, 28]               0
          Conv2d-121          [-1, 128, 28, 28]         131,072
     BatchNorm2d-122          [-1, 128, 28, 28]             256
    EdgeResidual-123          [-1, 128, 28, 28]               0
          Conv2d-124         [-1, 1024, 28, 28]       1,179,648
     BatchNorm2d-125         [-1, 1024, 28, 28]           2,048
           ReLU6-126         [-1, 1024, 28, 28]               0
        Identity-127         [-1, 1024, 28, 28]               0
          Conv2d-128          [-1, 128, 28, 28]         131,072
     BatchNorm2d-129          [-1, 128, 28, 28]             256
    EdgeResidual-130          [-1, 128, 28, 28]               0
          Conv2d-131          [-1, 512, 14, 14]         589,824
     BatchNorm2d-132          [-1, 512, 14, 14]           1,024
           ReLU6-133          [-1, 512, 14, 14]               0
        Identity-134          [-1, 512, 14, 14]               0
          Conv2d-135          [-1, 128, 14, 14]          65,536
     BatchNorm2d-136          [-1, 128, 14, 14]             256
    EdgeResidual-137          [-1, 128, 14, 14]               0
          Conv2d-138          [-1, 512, 14, 14]         589,824
     BatchNorm2d-139          [-1, 512, 14, 14]           1,024
           ReLU6-140          [-1, 512, 14, 14]               0
        Identity-141          [-1, 512, 14, 14]               0
          Conv2d-142          [-1, 128, 14, 14]          65,536
     BatchNorm2d-143          [-1, 128, 14, 14]             256
    EdgeResidual-144          [-1, 128, 14, 14]               0
          Conv2d-145          [-1, 512, 14, 14]         589,824
     BatchNorm2d-146          [-1, 512, 14, 14]           1,024
           ReLU6-147          [-1, 512, 14, 14]               0
        Identity-148          [-1, 512, 14, 14]               0
          Conv2d-149          [-1, 128, 14, 14]          65,536
     BatchNorm2d-150          [-1, 128, 14, 14]             256
    EdgeResidual-151          [-1, 128, 14, 14]               0
          Conv2d-152          [-1, 512, 14, 14]         589,824
     BatchNorm2d-153          [-1, 512, 14, 14]           1,024
           ReLU6-154          [-1, 512, 14, 14]               0
        Identity-155          [-1, 512, 14, 14]               0
          Conv2d-156          [-1, 128, 14, 14]          65,536
     BatchNorm2d-157          [-1, 128, 14, 14]             256
    EdgeResidual-158          [-1, 128, 14, 14]               0
          Conv2d-159         [-1, 1024, 14, 14]         131,072
     BatchNorm2d-160         [-1, 1024, 14, 14]           2,048
            ReLU-161         [-1, 1024, 14, 14]               0
          Conv2d-162         [-1, 1024, 14, 14]           9,216
     BatchNorm2d-163         [-1, 1024, 14, 14]           2,048
            ReLU-164         [-1, 1024, 14, 14]               0
        Identity-165         [-1, 1024, 14, 14]               0
          Conv2d-166          [-1, 384, 14, 14]         393,216
     BatchNorm2d-167          [-1, 384, 14, 14]             768
     InvertedResidual-168          [-1, 384, 14, 14]               0`

Total params: 11,690,672
Trainable params: 11,690,672
Non-trainable params: 0
----------------------------------------------------------------
Input size (MB): 2.23
Forward/backward pass size (MB): 527.29
Params size (MB): 44.60
Estimated Total Size (MB): 574.11

The original paper says the params in all the platforms are less than 10M.

**Comment by novice03** [comment]
Hello, I am taking a look into this. Can you please let me know where the paper says the params are less than 10M?

**Comment by taojianggit** [comment]
> Hello, I am taking a look into this. Can you please let me know where the paper says the params are less than 10M?

The table 1 in this paper [https://arxiv.org/abs/2004.14525v3](url) , please look at the last column.
![image](https://user-images.githubusercontent.com/26213651/137652623-8db970bd-6734-4c1c-983a-9cbd9ef0e486.png)
